apps/communications/dialer/test/unit/call_log_test.js

```diff
@@ -790,11 +790,30 @@ suite('dialer/call_log', function() {
       sinon.assert.notCalled(CallLog.renderEmptyCallLog);
     });
 
-    test('renderEmptyCallLog should be called when there is no missedgroup',
+    test('renderEmptyCallLog should be not called when there is no ' +
+         'missedgroup and it is in all call log view',
     function() {
       this.sinon.spy(CallLog, 'renderEmptyCallLog');
       CallLog.deleteLogGroups();
```

Comment:
Hi Yi-Fan,

Could we make this test look a bit more like the one below it where we append the missedGroup, (and leave it unfiltered/all vs. filtered) and the query for the missed call and set the checkbox to true and then delete(like you did in the test below).  I'm thinking that's closer to the failure case that you fixed.